flow/compositor_context.cc

```diff
@@ -135,7 +135,6 @@ RasterStatus CompositorContext::ScopedFrame::Raster(
     }
 
     if (needs_save_layer) {
-      FML_LOG(INFO) << "Using SaveLayer to protect non-readback surface";
```

Comment:
I'm curious why this is being deleted here.

Reply:
Style guide says logs should be actionable: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#only-log-actionable-messages-to-the-console

This one is not user actionable AFAICT. If there's something a developer could/should do with this information, we should make it clearer.